### DISCUSSION TITLE
How to migrate use of `LegacyCard.Section`?

### DISCUSSION THREAD

**Discussion by bjester** [Shopify/polaris-react#9787] [Q&A]
I looked through the [version 10 to 11 migration guide](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v10-to-v11.md) for the Polaris package, as well as the component documentation, and it isn't clear to me how to migrate to the new `Card` component for past usage of `Card.Section` structures?

For example, this taken from the documentation for what is now `LegacyCard`:
```jsx
    <LegacyCard title="Customer">
      <LegacyCard.Section>
        <p>John Smith</p>
      </LegacyCard.Section>
      <LegacyCard.Section title="Addresses">
        <LegacyCard.Subsection>
          123 First St
          <br />
          Somewhere
          <br />
          The Universe
        </LegacyCard.Subsection>
      </LegacyCard.Section>
    </LegacyCard>
```

**Comment by akospaska** [comment]
I am faced with the same issue ... :-(

**Comment by kyledurand** [comment]
Every card is purely composable now to allow builders to be more creative. The idea is that you use the Box component along with layout components like BlockStack / InlineStack to create layout within the card so there's no one way to migrate that we can document.

There are plenty of [examples](https://polaris.shopify.com/components/layout-and-structure/card?example=card-with-subdued-section) of how we use the new Card

Here's a [sandbox](https://codesandbox.io/p/sandbox/kind-cache-jc36ng?file=%2FApp.tsx%3A43%2C20) that closely matches the example you have above.

I'm closing this discussion for now. In the future please create issues over discussions for this sort of thing

**Reply by bjester** [reply]
> In the future please create issues over discussions for this sort of thing

![Screenshot from 2024-04-25 06-58-26](https://github.com/Shopify/polaris/assets/819838/ba7cc16d-8669-40bb-89ef-cc3b060fcb82)

I asked a question. In the future, update those guidelines or disable discussions if you don't want questions here. 

> there's no one way to migrate that we can document.

Oh I see, and that's a good reason to not document _at least one_ way in your migration guide. You're really inspiring people to use Polaris.

**Reply by akospaska** [reply]
In the documentation by the layout section the example code still uses a LegacyCard.Section.

When do you planning to update the documentation?

https://polaris.shopify.com/components/layout-and-structure/layout?example=layout-three-columns-with-equal-width

![image](https://github.com/Shopify/polaris/assets/82268269/2a2cdf91-fa1a-47be-8d9c-593cd90b2697)

**Reply by kyledurand** [reply]
@bjester this feels like more of an issue with documentation than a question about the system. 

@akospaska our team doesn't have the bandwidth to spend time on docs at the moment. I've added an [issue to our backlog](https://github.com/Shopify/polaris/issues/11948)
